uikit-dz/uikit-dz/Cells/ButtonNotificationTableViewCell.swift

```diff
@@ -0,0 +1,143 @@
+// ButtonNotificationTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Ячейка для постов с кнопками
+final class ButtonNotificationTableViewCell: UITableViewCell {
+    // перечисление для констант
+    enum Constant: String {
+        case story
+        case story2
+        case yourstory = "lavanda123 понравился ваш комментарий: \"Очень красиво!\" 12ч"
+        case ocean
+        case identifier = "ButtonNotificationTableViewCell"
+        case subscribe = "Подписаться"
+        case subscribed = "Вы подписаны"
+    }
+
+    // MARK: - Private properties
+
+    static let identifier = Constant.identifier.rawValue
+
+    private var buttonState: ButtonState = .disable {
+        didSet {
+            configureButtonState()
+        }
+    }
+
+    // MARK: - Private ui elements
+
+    private let profileImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: Constant.story.rawValue)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private let storyTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = Constant.yourstory.rawValue
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = .systemFont(ofSize: 12)
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private lazy var subscribeButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(Constant.subscribe.rawValue, for: .normal)
+        button.backgroundColor = .blue
+        button.titleLabel?.font = .systemFont(ofSize: 12, weight: .bold)
+        button.layer.cornerRadius = 10
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.addTarget(self, action: #selector(subscribeButtonPressed), for: .touchUpInside)
+        return button
+    }()
+
+    // MARK: - Initializer
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupUI()
+        setupConstraints()
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError()
+    }
+
+    // MARK: - Public methods
+
+    func configure(model: ButtonModel) {
+        profileImageView.image = UIImage(named: "\(model.profileImage)")
+        storyTitleLabel.text = model.text
+        buttonState = model.buttonState
+    }
+
+    // MARK: - Private methods
+
+    private func configureButtonState() {
+        switch buttonState {
+        case .enabled:
+            subscribeButton.setTitle(Constant.subscribe.rawValue, for: .normal)
+//            subscribeButton.isEnabled = true
```

Comment:
```suggestion
//            subscribeButton.isEnabled = true
```
такого не должно быть по проекту пройдись и удали